Src/Innovative.Blazor.Components/Components/Grid/InnovativeGrid.razor.css

```
@@ -1,8 +1,6 @@
 ::deep .rz-data-grid {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)